txscript/opcode.go

```diff
@@ -2510,6 +2510,10 @@ func opcodeCheckMultiSig(op *parsedOpcode, vm *Engine) error {
 				return err
 			}
 
+			if vm.hasFlag(ScriptVerifySchnorr) && len(signature) == 64 {
```

Comment:
should be len(signature) == 65     (64 for the sig + 1 for hashtype byte).

Reply:
Yeah right above it pops the sighash off so this will be 64 here.

Reply:
ah never mind, I see `signature` has hashtype already stripped off